app/views/checkout/thankyou.scala.html

```diff
@@ -81,9 +82,15 @@ <h3 class="section-divider">Register for an account</h3>
                     <div class="actions">
                         <button class="button button--primary button--large js-checkout-finish-account-submit">Confirm</button>
                     </div>
-
                 </form>
-            }
+                <div class="js-all-done" style="display: none">
```

Comment:
Use `is-hidden`. (If we need to do kind of contextual content more often we should possibly consider using client-side templates instead)